src/sass/_shame.scss

```diff
@@ -5,11 +5,12 @@
 
 // message while loading react app
 .loading-message {
-  text-align: center;
   padding: 2em 0;
+  text-align: center;
 }
 
-body .row {
+
+.row {
```

Comment:
Are all the changes in here necessary? They make me a little nervous, since they'll apply to all pages on vets.gov.